salt/_modules/metalk8s_monitoring.py

```diff
@@ -164,42 +162,45 @@ def get_alerts(state=None, **kwargs):
 
 
 def _requests_alertmanager_api(route, method="GET", **kwargs):
-    endpoint = __salt__["metalk8s_kubernetes.get_service_endpoints"](
+    endpoints = __salt__["metalk8s_kubernetes.get_service_endpoints"](
         "prometheus-operator-alertmanager",
         "metalk8s-monitoring",
         kwargs.pop("kubeconfig", None),
     )
```

Comment:
I know it's was already the case before, but can we use the service IP instead it would be cleaner to me, since here we will always it the first endpoint

Reply:
Changed.